### ISSUE TITLE
Find more efficient way to make semantic labels binary

### ISSUE THREAD

**Issue by a1302z** [TUM-AIMED/PrivateSegmentation#4]
Currently done by:

`transforms.Lambda(
                        lambda x: np.where(
                            x > 0.0,
                            torch.ones_like(x),
                            torch.zeros_like(
                                x
                            ),
                        )
                    )`

**Comment by NiWaRe** [comment]
Didn't see yours sry, added some more info to mine #7
